src/Request/Data/InvoiceProfile/InvoiceProfileData.php

```diff
@@ -266,17 +282,19 @@ public function setStreet2(string $street2): InvoiceProfileData
     }
 
     /**
-     * @param string $currencyCode
+     * @param CurrencyEnum $currencyCode
+     *
      * @return $this
      */
-    public function setCurrencyCode(string $currencyCode): InvoiceProfileData
+    public function setCurrencyCode(CurrencyEnum $currencyCode): InvoiceProfileData
```

Comment:
To avoid breaking change, deprecate the method, create a new one that is setCurrencyCodeEnum